### DISCUSSION TITLE
Supabase UI - Flutter support

### DISCUSSION THREAD

**Discussion by alon710** [supabase/supabase#2274] [Questions] [answered]
Hey!

Supabase UI looks **GREAT** and I wanted to know if there are any plans to support other technologies such as Flutter in the future?

Thanks!

**Comment by himeshp** [comment]
Database library works great.
Auth packages needs be more friendly, currently the Google sign in pop up opens in web view which is not that great experience.

I am a flutter developer myself and all of my projects needs ecosystem of auth and database.

**Comment by inian** [comment, marked as answer]
Hi @alon710, @phamhieu is working on Flutter support here - https://github.com/supabase/supabase-flutter
